app/models/framework/definition/RM858.fdl

```diff
@@ -0,0 +1,66 @@
+Framework RM858 {
+  Name 'Pan Govt Vehicle Leasing & Fleet Outsource Solutio'
+
+  ManagementCharge varies_by 'Spend Code' {
+    'Lease Rental'         -> 0.5%
```

Comment:
Is there a reason you chose "->" over what I would have thought would be a more intuitive "="?

Reply:
Two reasons: 1, we didn't want to imply equality or assignment anywhere, we wanted this to look like a map – when *this*, then *that* – and 2, if we do that we get to reuse this in dependent field validations as per the [spike](https://github.com/rgarner/parslet-spike/blob/master/run_parslet.rb#L39-L42).